### ISSUE TITLE
[Subtask]: Aggregate logging for prepared statement 

### ISSUE THREAD

**Issue by fengttt** [matrixorigin/matrixone#9834]
### Parent Issue

#8196

### Detail of Subtask

For prepared statement, we should log the prepare statement itself as usual, then for execution of the prepared statement, we will generate one log record per session per second (or 10 seconds, whatever is appropriate), saying that the prepared statement (uuid) was executed N times in this period, with X errors.

For DN, very short transactions too -- we should simply report there are N short transactions out of which X committed, Y abborted.

**Comment by gavinyue** [comment]
Working on this today to have an initial draft PR.

**Comment by gavinyue** [comment]
今天将做出修改

**Comment by fengttt** [comment]
in fact we should do these for short queries in general so we log every session/10 sec, there are so many short queries ...
maybe like 20 begins, 20 commits, 123 inserts, 3333 select short queries, and their accumulate running time.

Means we may add a field to statement_info,  for count, and accumulate run time.

**Comment by gavinyue** [comment]
Confirmed design with pm and start to implement

**Comment by gavinyue** [comment]
Working on the implementation.  Spent some time researching the design of window func in some other streaming lib. 

Will keep the existing batch-processor micro-batch design, but make aggregation using streaming design. 

ETA, june 12

**Comment by gavinyue** [comment]
https://github.com/gavinyue/matrixone/pull/9

Create the initial implementation after checking different options. Not be able to refactor the batch_processor pipeline. 

Need to do it incrementally.  So still use the existing micro-batch-based pipeline.

**Comment by gavinyue** [comment]
Pending PR merge

**Comment by gavinyue** [comment]
blocked on the merge run. still investigating

**Comment by gavinyue** [comment]
fix the load issue and tpcc hung problem 

resubmitted the pr

**Comment by gavinyue** [comment]
Pending for the pr merge. Tpcc failed.

**Comment by gavinyue** [comment]
Pending for the stmt_aggr pr merge. Stuck on dn oom

**Comment by heni02** [comment]
wait test,reopen

**Comment by heni02** [comment]
test done，https://github.com/matrixorigin/mo-nightly-regression/issues/66

and There are still missing testing points that need to be supplemented in communication with the product

**Comment by heni02** [comment]
After discussion, missing testing points  can be postponed to the next version

**Comment by heni02** [comment]
test done，https://github.com/matrixorigin/MO-Cloud/issues/1150

**Comment by heni02** [comment]
遗漏的bug还未解决完：#10663 #10551

**Comment by heni02** [comment]
bug已经解决
